instruments/abstract_instruments/instrument.py

```diff
@@ -671,10 +668,6 @@ def open_usb(cls, vid, pid):
         :return: Object representing the connected instrument.
         """
         # pylint: disable=no-member
-        if usb is None:
-            raise ImportError("USB support not imported. Do you have PyUSB "
-                              "version 1.0 or later?")
```

Comment:
`pyusb` is also a requirement for IK now. Should also add `>=1.0` to requirements.

Reply:
added, see `setup.cfg`